src/app/modules/ix-table/components/ix-table-body/cells/ix-cell-state-button/ix-cell-state-button.component.ts

```diff
@@ -31,66 +35,63 @@ interface RowState {
   styleUrls: ['./ix-cell-state-button.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class IxCellStateButtonComponent<T> extends ColumnComponent<T> {
+export class IxCellStateButtonComponent<T> extends ColumnComponent<T> implements OnInit {
   matDialog: MatDialog = inject(MatDialog);
   translate: TranslateService = inject(TranslateService);
   dialogService: DialogService = inject(DialogService);
   errorHandler: ErrorHandlerService = inject(ErrorHandlerService);
 
+  private cdr = inject(ChangeDetectorRef);
   private store$: Store<JobSlice> = inject<Store<JobSlice>>(Store<JobSlice>);
+  job = signal<Job>(null);
+  jobUpdates$: Observable<Job<ApiJobResponse<ApiJobMethod>>>;
+
+  ngOnInit(): void {
+    this.jobUpdates$ = this.store$.pipe(
+      select(selectJob((this.row as Job).id)),
```

Comment:
How is the `row` a `Job` here? If I have a replication task for example, then logically the row should be a replication task, while job should be some field within this replication task.